main.go

```diff
@@ -48,7 +48,7 @@ var (
 			},
 			Namespace: []string{
 				"https://storage.googleapis.com/riff-releases/latest/riff-build.yaml",
-				"https://storage.googleapis.com/riff-releases/riff-cnb-buildtemplate-0.1.0.pre.3.yaml",
+				"https://raw.githubusercontent.com/projectriff/riff-buildpack-group/cf202c43b5695a74185929e393e5c1005e6f1f91/riff-cnb-buildtemplate.yaml",
```

Comment:
We should keep release files like this on GCS, I uploaded the file as

```suggestion
				"https://storage.googleapis.com/riff-releases/riff-cnb-buildtemplate-0.1.1.pre.1.yaml",
```

Reply:
Agreed. This is really to unstuck the whole thing for now.
As a matter of fact, the CI for the buildpack group could reference timestamped tagged versions of the docker image (and push that to GCS)